doc/reference_manual/transport_model.tex

```diff
@@ -73,9 +73,11 @@ \section{Transport of substances}
 \end{align*}
 where $\vc n$ stands for the unit outward normal vector to $\partial\Omega_d$.
 We prescribe the following boundary conditions:
-On the inflow parts $\Gamma_I^+\cup\Gamma_D$, the user must provide \hyperA{SoluteTransport-DG-Data::bc-conc}{Dirichlet boundary condition} $c_D^i$ for concentrations:
-$$ c^i = c^i_D \mbox{ on }\Gamma_I^+\cup\Gamma_D. $$
-On $\Gamma_I^-$ we impose homogeneous Neumann boundary condition:
+On $\Gamma_D$, the user must provide \hyperA{SoluteTransport-DG-Data::bc-conc}{Dirichlet boundary condition} $c_D^i$ for concentrations:
+$$ c^i = c^i_D \mbox{ on }\Gamma_D. $$
+On the inflow $\Gamma_I^+$ the \hyperA{SoluteTransport-DG-Data::bc-conc}{reference concentration} $c_D^i$ is enforced through total flux:
```

Comment:
We have to distinguish inflow and out flow, since on the outflow part of boundary $q.n>0$ we have to use total flux equal to $q c_i$ (kind of upwind).